boost-common/src/main/java/io/openliberty/boost/utils/SpringBootUtil.java

```diff
@@ -159,6 +172,33 @@ public static Properties getSpringBootServerProperties(String buildDir) throws I
         String serverAddress = (String) allProperties.getOrDefault(SERVER_ADDRESS, DEFAULT_SERVER_ADDRESS);
         serverProperties.setProperty(SERVER_PORT, serverPort);
         serverProperties.setProperty(SERVER_ADDRESS, serverAddress);
+        
+        String serverSslKeystore = (String) allProperties.getProperty(SERVER_SSL_KEYSTORE);
+        String serverSslKeystorePassword = (String) allProperties.getProperty(SERVER_SSL_KEYSTORE_PASSWORD);
+        
+        if (serverSslKeystore != null && serverSslKeystorePassword != null ) {
+            serverProperties.setProperty(SERVER_SSL_KEYSTORE, serverSslKeystore);
+            serverProperties.setProperty(SERVER_SSL_KEYSTORE_PASSWORD, serverSslKeystorePassword);
+            
+            // Add any additional SSL properties
+            String serverSslKeystoreType = (String) allProperties.getProperty(SERVER_SSL_KEYSTORE_TYPE);
+            String serverSslKeystoreProvider = (String) allProperties.getProperty(SERVER_SSL_KEYSTORE_PROVIDER);
+            String serverSslKeyAlias = (String) allProperties.getOrDefault(SERVER_SSL_KEY_ALIAS, "default");
+            String serverSslKeyPassword = (String) allProperties.getProperty(SERVER_SSL_KEY_PASSWORD);
+            
+            if (serverSslKeystoreType != null) {
+                serverProperties.setProperty(SERVER_SSL_KEYSTORE_TYPE, serverSslKeystoreType);
+            }
+            if (serverSslKeystoreProvider != null) {
+                serverProperties.setProperty(SERVER_SSL_KEYSTORE_PROVIDER, serverSslKeystoreProvider);
+            }
+            
+            
+            if (serverSslKeyPassword != null) {
+                serverProperties.setProperty(SERVER_SSL_KEY_PASSWORD, serverSslKeyPassword);
+                serverProperties.setProperty(SERVER_SSL_KEY_ALIAS, serverSslKeyAlias);
```

Comment:
Why don't you check for the alias != null before setting this property? You do for all the others.

Reply:
So from my basic understanding of how this works, our "keyEntry" element in server.xml requires a password, but does not necessarily need an alias. In that case, the "name" attribute of "keyEntry" is "default."  When we are retrieving the spring boot properties, if "server-ssl-key-alias" is not provided, we default it to "default" so it will never be null at this point. If there is a password, we create this "keyEntry" with password and name.  If password isnt present, we do nothing.